src/routes/save-metadata.ts

```diff
@@ -6,12 +6,29 @@ interface EndpointDependencies {
   saveMetadata: SaveMetadata;
 }
 
+const isStringOrArray = (obj) => {
+  return (
+    typeof obj === 'string' ||
+    (obj.constructor === Array && obj.every((i) => typeof i === 'string'))
+  );
+};
+
 const createEndpoint = ({
   saveMetadata,
 }: EndpointDependencies): RouteOptions => ({
   method: 'POST',
   url: '/metadata',
   handler: async (req, reply) => {
+    const metadata = req.body;
+
+    for (const [key, value] of Object.entries(metadata)) {
```

Comment:
If you don't need the key, you can use `Object.values` instead.

Reply:
Ah, awesome!